### PR TITLE
Updated the markup(#1y2ract)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

Screenshot from 2022-01-19 13-32-56

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)
- [ ] I have changed the target to hacktoberfest branch

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -108,14 +108,18 @@ export default defineComponent({
...
       store,
       router
     }
   }
 });
 </script>
-
+<style scoped>
+ion-label[slot="end"] {
+   text-align: end;
```
Comment (azkyakhan) [inline src/views/Settings.vue:123]: improve indentation

**Review by azkyakhan — CHANGES_REQUESTED** [review]

File: src/views/Settings.vue (hunk 1 above)
Comment (disha1202) [reply src/views/Settings.vue]: Improved indentatiom

**Comment by azkyakhan** [comment]
LGTM

File: src/views/Settings.vue [hunk 2]
```diff
@@ -21,7 +21,7 @@
         <ion-item>
           <ion-icon :icon="codeWorkingOutline" slot="start"/>
           <ion-label>{{ $t("OMS") }}</ion-label>
-          <ion-note slot="end">{{ instanceUrl }}</ion-note>
+          <ion-label slot="end">{{ instanceUrl }}</ion-label>
```
Comment (azkyakhan) [inline src/views/Settings.vue:24]: LGTM

**Review by azkyakhan — APPROVED** [review]

**Pull request merged**